src/gui/mzroll/database.cpp

```diff
@@ -570,6 +571,10 @@ int Database::loadCompoundCSVFile(string filename){
             for(int i=0; i < categorylist.size(); i++) compound->category.push_back(categorylist[i]);
             if (addCompound(compound))
                 loadCount++;
+        } else {
+            if (!name.empty())
```

Comment:
@saifulbkhan why check for name not being empty? id will be set to name if name is available anyway.

Reply:
So usually ID is something like "cmpd21" or something, but if the name is empty it will become an empty string and the list of discarded compounds will look like: 

- UMP
- ATP
- 
- Succinate

I thought giving a compound ID would make more sense at least than letting it be blank.

Reply:
@saifulbkhan Got it!

Reply:
We will soon work on merging these two database files. I recently opened an [issue](https://github.com/ElucidataInc/ElMaven/issues/869) for this.